src/components/ImportJobs/components/FileUploader/readme.md

```diff
@@ -0,0 +1,117 @@
+# FileUploader
+
+Component built on top of [ReactDropzone](https://react-dropzone.netlify.com) component. Provides drag-and-drop zone for file uploading. Note that uploading files to server is out of the scope of this component and should be implemented additionally on client and server.
+
+## Basic Usage
+
+```javascript
+<FileUploader
+  title="Drop to start import"
+  uploadBtnText="or choose files"
+  isDropZoneActive={isDropZoneActive}
+  onDrop={onDrop}
+/>
+```
+
+## onDrop callback example
+
+onDrop callback takes an array of accepted files as the first argument and an array of rejected files as the second argument. Files are being accepted or rejected based on [`accept`](https://react-dropzone.netlify.com/#src) prop.
+
+Example of onDrop method that validates files based on their extensions with some custom logic.
+
+```javascript
+/**
+  * @param  {Array<File>} acceptedFiles
+  * @param  {Array<File>} rejectedFiles
+  */
+onDrop = (acceptedFiles, rejectedFiles) => {
+  const isValidFileExtensions = this.validateFileExtensions(acceptedFiles);
+
+  if (isValidFileExtensions) {
+    this.setState({
+      isDropZoneActive: false,
+      redirect: true,
+      acceptedFiles,
+      rejectedFiles,
+    });
+
+    return;
```

Comment:
Do you consider empty return as a good solution?

Reply:
That was Victor's suggestion to have it like that.

Reply:
Yep, that main idea is to avoid `if` `else` statements in case if `else` statement is the last one.